account_credit_control/README.rst

```diff
@@ -1,11 +1,16 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
Please, change all headers of all files by this new format:

https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3